### DISCUSSION TITLE
How to set the keyboard shortcut to use Windows key?

### DISCUSSION THREAD

**Discussion by nhan000** [theJayTea/WritingTools#59] [Q&A] [answered]
I tried `win+F6` and `windows+F6` and neither worked. Can the `Win` key be use in the keyboard shortcut or am I doing something wrong? 

Thanks a lot for this app :)

**Comment by Seviks** [comment, marked as answer]
This works for me:
`cmd_l`

Source:  https://pythonhosted.org/pynput/keyboard.html#pynput.keyboard.Key.cmd

> cmd_l = <Key.f1: 0>
>    The left command button. On PC platforms, this corresponds to the Super key or Windows key, and on Mac it corresponds to the Command key. This may be a modifier.

**Reply by nhan000** [reply]
Thanks so much. To much it a bit clearer, windows key is `cmd_l`, so if you want `windows+F6` like me, put `cmd_l+F6` as the shortcut key.
